docs/api/utility.rst

```diff
@@ -215,21 +215,40 @@ command but doesn't throw an error if the list is empty or not defined.
     set(mylist A B A)
     blt_list_remove_duplicates( TO mylist )
 
+.. _blt_find_target_dependencies:
+
+blt_find_target_dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    set(_target_list "")
+    blt_find_target_dependencies(TARGET <target> TLIST _target_list)
+
+Recursively adds all link libraries and interface link libraries of the given
+target to the given list. Handles CMake targets and BLT registered libraries.
+
 .. _blt_convert_to_system_includes:
 
 blt_convert_to_system_includes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: cmake
 
-    blt_convert_to_system_includes(TARGET <target>)
+    blt_convert_to_system_includes(TARGET   <target>
+                                   TARGETS  [<target>...]
+                                   CHILDREN [TRUE | FALSE (default)]
+                                   [QUIET])
 
```

Comment:
The docs should also have a comment/note that `TARGET` will be deprecated in the near future, so it would be better to use `TARGETS`